test/shim_env_command.bats

```diff
@@ -80,5 +80,5 @@ teardown() {
 
   # Should not contain duplicate colon
   run grep '::' <(echo "$path_line")
-  [ "$duplicate_colon" == "" ]
```

Comment:
Another bug. This variable was never set so this line wasn't actually testing anything.